runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,132 @@
 
 /* @ddr_namespace: map_to_type=J9JavaAccessFlags */
 
-/* Constants from J9JavaAccessFlags */
+/* ROM class modifiers */
```

Comment:
These don't just apply to ROM _classes_ (i.e. `J9ROMClass.modifiers`): some (e.g. `J9AccSynthetic`) also apply to fields
(e.g. `J9ROMFieldShape.modifiers`) and methods (e.g. `J9ROMMethod.modifiers`). We don't want to replacing missing comments with misleading ones. I think what we're aiming for is to group these constants according to whether they apply to the `modifiers` field (of a class, field or method) or to the `extraModifiers` field of a class.

Reply:
Good point, none of these are really complete lists. I've updated the comments to clarify this.